etc/calplus/calplus.conf.sample

```diff
@@ -0,0 +1,210 @@
+[DEFAULT]
```

Comment:
no, look carefully on why glance need to put sample config file there and take a look around nova, cinder... 
I prefer delete it on our project.